### ISSUE TITLE
Download Patch

### ISSUE THREAD

**Issue by sfobrian** [MehediH/Tweeten#303]
Won't let me set a download path. 
When selecting my desktop, for example, "set download path" remains greyed out.  Won't let me set and save.

**Comment by jaydeflix** [comment]
fyi, dupe of 283, 290, and 299. 

283 has the core issue. Looks like @MehediH should (I think) call the FolderBrowserDialog instead of SaveFileDialog.

**Comment by sfobrian** [comment]
Yeah I saw others had reported the same issue.  But it's been a few months now and no fix yet.
